### DISCUSSION TITLE
3x Fans on Fan Hub, show up at 0RPM in FanControl

### DISCUSSION THREAD

**Discussion by cantodorum** [Rem0o/FanControl.Releases#2810] [General]
I've got 3x the same NF-A14 plugged into a fan hub built-in to my case. It gets supplementary 12V power and I then connected the cable coming from the fan hub to my MB header "CHA_FAN2".
I got two of the fans linked via a Y-splitter from noctua, which then plugs into the hub, while the other fan is just on an extension cable going directly into the hub.
I tried plugging the pair into "CHA_FAN2" directly and the other one in "M.2_FAN", but then this one is running at max speed all the time.
![grafik](https://github.com/user-attachments/assets/2c61eeab-135a-415b-a1ad-4fd3d1f43315)
I got my fan profiles for all fan headers set to MANUAL in the BIOS and put in a curve thats just static 20%, so that FanControl can take all of the control.

I got an ASUS X570-E MB.
![2024 11 01_17-12-25_NVIDIA_Overlay](https://github.com/user-attachments/assets/ac836fc7-ecf8-41d7-b37e-9a418ef4a4be)

**Comment by Fr0stX76** [comment]
Hi, I suspect a mix of 3 pin (dc) vs 4 pin (pwm) here.

Are your fans/hub/y-splitter/extension all 4 pins? Also what is your bios set at, dc or pwm?

If everything is 4 pin, set your bios to pwm and 50% for the channels you want fc to control.

If everything is 3 pins (and bios in dc), with the hub being fed supplementary 12V power, it will not work in dc control scenario (since the hub is 12v fixed). You will need to remove the hub for dc to work.

If either the hub, the y-splitter or the extension are 3 pins, with 4 pin fans, it will not work in pwm control scenario.

About the 0% , most hubs have one channel that passthrough rpm feedback from the fan down to the mb. It usually has a mark or is of a different color on the hub. You need to connect at least one fan to it in order for your mb header to get feedback. Splitters usually work.
